data/wiki/hello-world.md

```diff
@@ -761,16 +756,14 @@ reasonable default based on the target the user chooses to pass to
 `mirage configure` - just like the reasonable default provided by
 `generic_kv_ro` in the previous example.
 
-`generic_stackv4` attempts to build a sensible network stack on top of
-the physical interface given by `default_network`.  There are quite a
-few configuration keys exposed when `generic_stackv4` is given related
-to networking configuration. For a full list, try `mirage help
-configure` in the `device-usage/network` directory.
+`generic_stackv4v6` attempts to build a sensible network stack on top of
+the physical interface given by `default_network`.
```

Comment:
(This was already here before, so it's not directly related to this PR)

For me, "sensible"/"reasonable" is quite similar to "trivial" in the context of maths: Saying that something is "trivial", "sensible" or "reasonable" doesn't add information, it just makes the reader think that they don't understand. Another very good example in that context is the [API documentation of `default_network`](https://github.com/mirage/mirage/blob/main/lib/mirage.mli#L431-L432):
> [default_network] is a dynamic network implementation which attempts to do something reasonable based on the target.

I don't have enough knowledge to explain what a "sensible network stack" is, so the only suggestion I can give is to get rid of the word "sensible":
```suggestion
`generic_stackv4v6` builds a network stack on top of
the physical interface given by `default_network`.
```